lib/classes/CssTransformator.class.js

```diff
@@ -36,6 +37,7 @@ class CssTransformator {
             .join("|");
         // separate in regular expression
         // we will replace all instances of these pseudo selectors; hence global flag
+        // Not sure the next lines are correct
```

Comment:
What do you mean?

Reply:
1st - `this._PSUEDO_SELECTOR_REGEXP` - looks like a typo. `PSUEDO` instead of `PSEUDO`
2nd - i didn't find a place where you use it.

Reply:
True. Didn't find it either. Will remove it after merge